src/NewChain/CliqueGenesis.ts

```diff
@@ -0,0 +1,856 @@
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000e6': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000e7': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000e8': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000e9': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000ea': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000eb': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000ec': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000ed': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000ee': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000ef': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f0': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f1': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f2': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f3': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f4': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f5': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f6': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f7': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f8': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000f9': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000fa': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000fb': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000fc': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000fd': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000fe': {
+          balance: '0x1',
+        },
+        '00000000000000000000000000000000000000ff': {
+          balance: '0x1',
+        },
+      },
+      number: '0x0',
+      gasUsed: '0x0',
+      parentHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
+    }
+
+    // Remove leading `0x` as it should not be present in the genesis addresses.
+    deployer = CliqueGenesis.removeLeading0x(deployer);
+    sealer = CliqueGenesis.removeLeading0x(sealer);
+
+    // Equals 800 mio. in hex format. Is a given from the original OST EIP20 contract.
+    initialGenesis.alloc[deployer] = { balance: '0x295be96e640669720000000' };
```

Comment:
It is done by `peepeth`, the go-ethereum tool to generate genesis files. I was wondering the same, together with @benjaminbollen and @pgev. We are not certain why it is done, but we think it may have to do with "address reservation" of opcodes.

Reply:
This is related to ‘precompiled’ contracts

https://solidity.readthedocs.io/en/latest/units-and-global-variables.html#mathematical-and-cryptographic-functions
`
When running sha256, ripemd160 or ecrecover on a private blockchain, you might encounter Out-of-Gas. This is because these functions are implemented as “precompiled contracts” and only really exist after they receive the first message (although their contract code is hardcoded). Messages to non-existing contracts are more expensive and thus the execution might run into an Out-of-Gas error. A workaround for this problem is to first send Wei (1 for example) to each of the contracts before you use them in your actual contracts. This is not an issue on the main or test net.
`

Reply:
Thanks @deepesh-kn !